metricbeat/scripts/modules_collector.py

```diff
@@ -0,0 +1,48 @@
+import os
+import argparse
+import yaml
+import six
+
+# Collects module configs to modules.d
+
+
+def collect(docs_branch):
+
+    base_dir = "module"
+    path = os.path.abspath("module")
+
+    # TODO add module release status if beta or experimental
+    header = """# Module: {module}
+# Docs: https://www.elastic.co/guide/en/beats/metricbeat/{docs_branch}/metricbeat-module-{module}.html
+
+"""
+
+    # Create directory for module confs
```

Comment:
Could we have the docs link also in the reference config file?

Reply:
As this is the module collection only for metricbeat, why did it change the other config files?

Reply:
I was talking about reference file, which is being collected by a different script